setup.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash 
+
```

Comment:
I may add a check here for conda to make sure the user has it (I know, I'm not great at making sure dependencies exist as pointed out by your recent issue).
something like
```
if [ -z $(which conda) ]; then
    echo "please install anaconda, see here: https://www.anaconda.com/download/"
    exit 1
fi
```